spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/Unset.scala

```diff
@@ -0,0 +1,43 @@
+package com.datastax.spark.connector.types
```

Comment:
I suppose the file should be named `CassandraOption.scala` rather than `Unset.scala`